testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_preserve_floodplain.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import netCDF4 as nc4
+import sys
+
+if __name__ == "__main__":
+
+    mesh_file = sys.argv[1]
+    value = sys.argv[2]
```

Comment:
I'm thinking we want this to be done with argparse so that the script can return some argument help when run.  But, given that it is a small and short script this may not be needed.  The challenge here is that if it grows larger than about 10 lines in the future using `sys.argv` will be a detriment to clarity.

Reply:
I agree, I switched to argparse in the latest commit.